src/repository/opamVCS.mli

```diff
@@ -29,7 +29,7 @@ module type VCS = sig
       Be aware that the remote URL might have been changed, so make sure
       to update accordingly. *)
   val fetch:
-    ?cache_dir:dirname -> ?subpath:subpath -> dirname -> url ->
+    ?full_fetch:bool -> ?cache_dir:dirname -> ?subpath:subpath -> dirname -> url ->
```

Comment:
It is missing documentation about the new argument.

Reply:
The optional arguments like `cache_dir` and `subpath` wasn't added, this is why.